cfpb/cfgov-refresh
# Don't load nonresponsive template header js in IE8

Header js causes error in IE8


## Changes

- Add conditional comment to `base_nonresponsive.html` so `header.nonresponsive.js` doesn't load in IE8

## Testing

- Check out branch
- Generate atomic js files: gulp scripts:ondemand  
- Navigate to http://localhost:8000/find-a-housing-counselor/ in IE8 and make sure there is no header-related error
- Navigate to http://localhost:8000/find-a-housing-counselor/ in modern browser & ensure that header still works

## Review

- @cfpb/cfgov-frontends 



## Checklist
